### PR TITLE
Update the README file for the master branch

### PR DESCRIPTION
**Pull request by janette**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,103 +31,106 @@ export PATH=$PATH:/myworkspace/dkan-tools/bin
...
+4. Get Drupal dependencies and DKAN modules. This will create the symlinks necesarry to create a working Drupal site under _/docroot_.
 
 ```bash
 dktl make
 ```
 
-The `make` command will get all of DKAN's dependencies _including_ Drupal core. It will also create all the symlinks necesarry to create a working Drupal site under _/docroot_.
+  - Flag options:
+      * If you are working directly on the DKAN project or one of its libraries and want to be able to commit changes and submit pull requests, use the `--prefer-source` option with `dktl make`. This option will be passed directly to Composer; see the [Composer CLI documentation](https://getcomposer.org/doc/03-cli.md#command-line-interface-commands) for more details.
+      * To install the React frontend application add the `--frontend` option with `dktl make`. This will download the app to `src/frontend` and symlink the files to `docroot/data-catalog-frontend`.
```
Comment (dharizza) [inline README.md:71]: Should we specify how to build the frontend? Maybe at least a link to that? Maybe not at all as it is not dkan-tools specific?

**Comment by janette** [comment]
excellent suggestions @dharizza ready for second review

File: README.md [hunk 2]
```diff
@@ -209,7 +190,7 @@ If you include this in your dktl.yml file, typing `dktl restore` without any arg
 
 ## Custom Commands
 
-Projects to can define their own commands. To create a custom command, create a new class inside of this project with a similar structure to the this one:
+Projects to can define their own commands. To create a custom command, add a file named `CustomCommands.php` and add it to `src/command/`, create a new class in the file with a similar structure to the this one:
```
Comment (dharizza) [inline README.md:193]: Remove the extra "the" at the end.

**Comment by janette** [comment]
@dharizza good catch thanks

**Pull request merged**